p2p/dial.go

```diff
@@ -335,7 +334,7 @@ func (d *dialScheduler) logStats() {
 	if d.lastStatsLog.Add(dialStatsLogInterval) > now {
 		return
 	}
-	if d.dialPeers < dialStatsPeerLimit && d.dialPeers < d.maxDialPeers {
```

Comment:
I think we should remove this. Without `dialStatsPeerLimit`, we will print "looking for peers" even when we aren't looking for peers. At best its noisy. At worst its untrue.

Reply:
IMO, just drop this commit

Reply:
dropped it.